pkgs/by-name/li/libfaketime/package.nix

```diff
@@ -18,9 +18,19 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-a0TjHYzwbkRQyvr9Sj/DqjgLBnE1Z8kjsTQxTfGqLjE=";
   };
 
-  patches = [
-    ./nix-store-date.patch
-  ];
+  patches =
+    [
+      ./nix-store-date.patch
+    ]
+    ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) [
+      # fix build on aarch64-darwin.
+      # ref. https://github.com/wolfcw/libfaketime/pull/497
+      # this was merged upstream an can be removed on next release
+      (fetchpatch {
+        url = "https://github.com/wolfcw/libfaketime/pull/497.patch";
```

Comment:
Isn't it better to fetch from the commit in the master branch? I think theoretically someone could re-open that PR and break the hash.

Reply:
true, but actually this is not fixing the issue, so maybe we don't even need this patch.
I messed my test, sorry for this.